modules/dcache-chimera/src/main/java/org/dcache/chimera/namespace/ChimeraCleaner.java

```diff
@@ -75,6 +75,9 @@
     private static final Logger _log =
         LoggerFactory.getLogger(ChimeraCleaner.class);
 
+    // ToDo: Spring Implementation for queryLimit ( used in runNotification() )
```

Comment:
Is this going to be added with another commit, or is it just an annotation for our future reference?